test/antithesis/cp-combined/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+FROM ubuntu:bionic-20200403
+
+RUN apt-get update \
+    && apt-get install -y ca-certificates \
+    && echo "deb [trusted=yes] https://packages.confluent.io/deb/6.1 stable main" >> /etc/apt/sources.list.d/confluent.list \
+    && apt-get update \
+    && apt-get install -y openjdk-11-jdk confluent-kafka confluent-schema-registry
```

Comment:
Do we need to pin the versions of those products to 5.5.3?

Reply:
Oops! Good catch!